app/component/ItinerarySummaryListContainer/components/components/Link.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import cx from 'classnames';
+
+const Link = ({ msgId, href }) => (
```

Comment:
We already use the Link component from found in a lot of places which is meant for links within this site. Should we name this something like ExternalLink so the difference between this and the found Link is more obvious. I was quite surprised there wasn't already a component like this somewhere. I would assume something like this would also be needed in other places. Is putting this component inside this ItinerarySummaryListContainer directory the right place? Is this generic enough so it could also be used elsewhere?

Reply:
Actually there already exists the component https://github.com/HSLdevcom/digitransit-ui/blob/v2/app/component/ExternalLink.js but it has a different DOM structure and CSS definitions which affect positioning. There are 18 external anchor tags in the code base (`<a>`) but almost all of them have individual styling.  I think it's better to substitute this back to `ChangeDepartureTimeLink` and `PastLink`.